doc/man/8/ceph.rst

```diff
@@ -1019,6 +1021,43 @@ Usage::
 
 	ceph osd pool stats {<name>}
 
+Subcommand ``application`` is used for adding an annotation to the given
+pool. By the default the applications are be object storage, block storage,
```

Comment:
> are be object storage

s/are/can/

Reply:
@tchaikov I reworded the entire paragraph to make it a bit more succinct. I hope you don't mind.  :)